frontend/src/components/RubricList/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { Link } from "react-router-dom";
+import { Table, TableCell, TableDiv, TableRow } from "../Table";
+import "./styles.css";
+import { FaChevronDown, FaChevronRight } from "react-icons/fa6";
+import Button from "../Button";
+import { useState } from "react";
+
+
+interface IRubricRowData extends React.HTMLProps<HTMLDivElement> {
+    rubricData: IFullRubric,
+}
+
+const RubricRow: React.FC<IRubricRowData> = ({
+    rubricData,
+}) => {
+    const [collapsed, setCollapsed] = useState(true);
+
+    return (
+        <>
+            <TableRow
+                className={!collapsed ? "TableRow--expanded" : undefined}
+                onClick={() => {
+                    setCollapsed(!collapsed);
+                }}
+            >
+                <TableCell>
+                    {collapsed ? <FaChevronRight /> : <FaChevronDown />}
+                </TableCell>
+                <TableCell>
+                    {rubricData.rubric.name}
+                </TableCell>
+                <TableCell>
+                <Link to={`/app/rubrics/new`} state={{ rubricData }}>
+                    <Button href="" variant="secondary" >
+                        Edit this Rubric
+                    </Button>
+                </Link>
+                </TableCell>
+            </TableRow>
+            {!collapsed && (
+                <TableDiv>
+                    <Table cols={3} className="ItemTable">
+                        {rubricData.rubric_items &&
+                            rubricData.rubric_items.map((item, i: number) => (
+                                <TableRow
```

Comment:
To avoid the first list item being a header row, ensure that the first row is the labels for each column. In this case, "Rubric Item" and "Points" respectively. This way all the actual rubric items will populate in the rows below and have the correct styling!